django_lightweight_queue/middleware/transaction.py

```diff
@@ -2,16 +2,13 @@
 
 class TransactionMiddleware(object):
     def process_job(self, job):
-        if not connection.in_atomic_block:
-            transaction.set_autocommit(False)
+        transaction.atomic(savepoint=False).__enter__()
```

Comment:
From the Django docs: https://docs.djangoproject.com/en/1.7/topics/db/transactions/#django.db.transaction.atomic

> You can disable the creation of savepoints for inner blocks by setting the savepoint argument to False. If an exception occurs, Django will perform the rollback when exiting the first parent block with a savepoint if there is one, and the outermost block otherwise. Atomicity is still guaranteed by the outer transaction. This option should only be used if the overhead of savepoints is noticeable. It has the drawback of breaking the error handling described above.

I'm not certain from reading this, maybe it should be True, I'm not sure performance is a concern here?